descriptor.md

```diff
@@ -36,7 +36,9 @@ The following fields contain the primary properties that constitute a Descriptor
 
 - **`urls`** *array*
 
-  This OPTIONAL property specifies a list of URLs from which this object MAY be downloaded.
+  This OPTIONAL property specifies a list of URIs from which this object MAY be downloaded.
+  Each entry MUST conform to [RFC 3986][rfc3986].
+  Entries SHOULD use the `http` and `https` schemes, as defined in [RFC 7230][rfc7230-s2.7].
```

Comment:
There's no danger of using a maliciously substituted blob, because you'll hash it when you get it.  Using HTTP just means detecting that sort of denial of service is more expensive.  And it means packet sniffers can see what you're asking for.  I expect there are situations where both of those are acceptable risks.